Makefile

```diff
@@ -17,6 +17,15 @@ integration-tests:
 	fi
 	@./integration-tmp/bin/bats ./test/integration/dnet
 
+./cmd/dnet/dnet:
+	make build
+
+clean:
```

Comment:
First of all since this is not a real target i.e not an actual output file you need to add this target in the `.PHONY` target list. Otherwise an empty unnecessary file called `clean` will be created.